quattor/functions/strings.pan

```diff
@@ -0,0 +1,29 @@
+declaration template quattor/functions/strings;
+
+include { 'pan/functions' };
+
+#version comparison function (should not be here)
+function comparever = {
+  if(ARGC!=2){
+    error("comparever: wrong number of arguments. Required 2, given "+to_string(ARGC)+".");
```

Comment:
use format, like 
```pan
error(format("comparever: wrong number of arguments. Required 2, given %s.", to_string(ARGV));
```
(this also prints the actual arguments, not only the number , to make clear what is passed)